ios/BT/API/Model/Exposure.swift

```diff
@@ -6,20 +6,11 @@ import RealmSwift
 class Exposure: Object, Codable {
   @objc dynamic var id: String = .default
   @objc dynamic var date: Int = 0
-  @objc dynamic var duration: Double = 0.0
```

Comment:
Hm, are these not needed anymore?

Reply:
the UI only makes use of the exposure's `date` property (the use of `duration` etc is an artifact of an outdated design spec for the UI). 
![IMG_497B415B1FD4-1](https://user-images.githubusercontent.com/2637355/92998089-e41cc400-f4e5-11ea-8bf4-48a0d41d406b.jpeg)